packages/rancher-webhook/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,10 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -4,6 +4,7 @@
+   catalog.cattle.io/namespace: cattle-system
+   catalog.cattle.io/os: linux
+   catalog.cattle.io/release-name: rancher-webhook
++  catalog.cattle.io/rancher-version: ">= 2.5.10 <2.5.99"
```

Comment:
this is a quick fix for adding the annotation. 
It will be moved to the original chart in the next release.